config/test.exs

```diff
@@ -54,3 +54,7 @@ config :code_corps,
 
 config :code_corps, :cloudex, CloudexTest
 config :cloudex, api_key: "test_key", secret: "test_secret", cloud_name: "test_cloud_name"
+
+# supply sample .pem with RSA_PRIVATE_KEY for tests
+config :code_corps,
+  github_app_pem: "./test/fixtures/github/app.pem" |> File.read!
```

Comment:
We have an actual PEM on CI, so I wonder if this shouldn't be an `||` statement somehow.